index.js

```diff
@@ -76,5 +76,15 @@ export default async function pRetry(input, options) {
 				}
 			}
 		});
+
+		if (options.signal && !options.signal.aborted) {
+			options.signal.addEventListener('abort', () => {
+				operation.stop();
+				const reason = options.signal.reason ?? new DOMException('The operation was aborted.');
```

Comment:
Seems like we can't use `??` if we want to keep compatibility with Node 12.

Reply:
Yes, I don't plan to target Node.js 14 yet.